statsmodels/tsa/vector_ar/var_model.py

```diff
@@ -657,26 +961,42 @@ def acorr(self, nlags=None):
 
     def plot_acorr(self, nlags=10, linewidth=8):
         "Plot theoretical autocorrelation function"
-        plotting.plot_full_acorr(self.acorr(nlags=nlags), linewidth=linewidth)
+        plotting.plot_full_acorr(self.acorr(nlags=nlags), linewidth=linewidth,
+                                 names=self.model.endog_names)
 
-    def forecast(self, y, steps):
-        """Produce linear minimum MSE forecasts for desired number of steps
-        ahead, using prior values y
+    def forecast(self, y=None, steps=1):
+        """Minimum MSE forecasts for desired number of steps
 
         Parameters
         ----------
-        y : ndarray (p x k)
+        y : ndarray (k_ar x neqs)
+            Deprecated. The initial values to use for the forecasts. If None,
+            the last k_ar values of the original endogenous variables are
```

Comment:
check if we really want to deprecate this.

What do we use for arbitrary prediction? i.e. independent of an estimated model, just given params.